src/luascript.cpp

```diff
@@ -9933,6 +9917,19 @@ int LuaScriptInterface::luaGroupGetMaxVipEntries(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaGroupHasFlag(lua_State* L)
+{
+	// group:hasFlag(flag)
+	Group* group = getUserdata<Group>(L, 1);
+	if (group) {
+		uint64_t flag = getNumber<uint64_t>(L, 2);
+		pushBoolean(L, group->flags & flag);
```

Comment:
this implicit conversion I'm sure that at least VC will complain about it, furthermore it isn't performance friendly